easycv/models/ocr/heads/cls_head.py

```diff
@@ -0,0 +1,24 @@
+# Modified from https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/ppocr/modeling/heads/cls_head.py
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class ClsHead(nn.Module):
```

Comment:
register this head and use other name

by the way, it seems to be the same as common classification head, reuse the ClsHead in classification if it can

Reply:
done